projects/VersionSuffixWithoutPrefix/VersionSuffixWithoutPrefix.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should there be a separate one for only Version as well?
Also I think I would just call this VersionSuffix.cs (including folder)

Reply:
Only version is part of `CompliantCSharpPackage`